### ISSUE TITLE
Could not find any version that matches com.google.firebase:firebase-crashlytics:[15.0.0, 16.0.0)

### ISSUE THREAD

**Issue by asfdfdfd** [OneSignal/OneSignal-Gradle-Plugin#123]
I've added OneSignal Gradle Plugin 0.12.7 today and get sync error.

```
ERROR: Unable to resolve dependency for ':app@mirrorProductionDebugAndroidTest/compileClasspath': Could not find any version that matches com.google.firebase:firebase-crashlytics:[15.0.0, 16.0.0)

ERROR: Unable to resolve dependency for ':app@mirrorProductionDebug/compileClasspath': Could not find any version that matches com.google.firebase:firebase-crashlytics:[15.0.0, 16.0.0)
```

I'm not sure that Firebase Crashlytics [has 15.0.0 or 16.0.0 version at all].(https://mvnrepository.com/artifact/com.google.firebase/firebase-crashlytics)

**Comment by jkasten2** [comment]
@asfdfdfd Can you share your `build.gradle`, `app/build.gradle`, and your Gradle version? We will need to this to reproduce the issue.

**Comment by jkasten2** [comment]
@asfdfdfd We have identified that this plugin does not consider Gradle BOM versions when picking a version for a dependency which I believe is what you are seeing here.

I have opened issue #127 to track the missing BOM support.

As a work around you set a specific `firebase-crashlytics` version like this in your `app/build.gradle`.
```gradle
dependencies {
    // ...
    implementation 'com.google.firebase:firebase-crashlytics:17.0.1'
}

**Comment by jkasten2** [comment]
@asfdfdfd This is now fixed in the [0.12.8 release](https://github.com/OneSignal/OneSignal-Gradle-Plugin/releases/tag/0.12.8)
